packages/app-utils/src/components/AppBootstrap.tsx

```diff
@@ -56,6 +66,25 @@ export function AppBootstrap({
     });
   }, []);
   useBroadcastLoginListener(onLogin, onLogout);
+
+  const contextActions = [
+    {
+      // Exporting logs in a general context
+      action: () => {
+        exportLogs(
+          logHistory,
+          {
+            uiVersion,
+            userAgent: navigator.userAgent,
+          },
+          store.getState()
+        );
+      },
+      shortcut: GLOBAL_SHORTCUTS.EXPORT_LOGS,
+      isGlobal: true,
+    },
+  ];
```

Comment:
I had mentioned on our 1:1 that the action was only in two places, so we didn't need to factor it out again. But now we've got it in a few places, I'd make a method in the `app-utils` package to create the context action.

```typescript
function createExportLogsContextAction(metadata?: ..., reduxData?: ...) {
  ...
}
```

We should also memoize the `contextActions` here using `useMemo` so we're not re-rendering unnecessarily.